internal/exec/terraform.go

```
@@ -437,7 +437,7 @@ func ExecuteTerraform(info schema.ConfigAndStacksInfo) error {
 
 	// Run `terraform workspace` before executing other terraform commands
 	// only if the `TF_WORKSPACE` environment variable is not set by the caller
-	if info.SubCommand != "init" && !(info.SubCommand == "workspace" && info.SubCommand2 != "") {
+	if info.SubCommand != "init" && info.ComponentBackendType != "http"  && !(info.SubCommand == "workspace" && info.SubCommand2 != "") {
```

golangci-lint (1.64.8)

[error] File is not properly formatted

(gofumpt)